lib/kernel/sleef/arch/helpervecext.h

```diff
@@ -853,3 +853,16 @@ static INLINE void vscatter2_v_p_i_i_vq(Sleef_quad *ptr, int offset, int step, v
 
 static INLINE void vsscatter2_v_p_i_i_vq(Sleef_quad *ptr, int offset, int step, vquad v) { vscatter2_v_p_i_i_vq(ptr, offset, step, v); }
 #endif
+
+
+// TODO
```

Comment:
Is it nice to have such dummy functions committed?

Reply:
well, SLEEF is done in a way where you have the main files with algorithms using helpers (e.g. ``vadd_vf_vf_vf``, and based on the CPU type they include a "helper header" which translates helpers to actual intrinsics (``vadd_vf_vf_vf`` -> ``_mm256_add_ps`` on AVX2, etc). So i must include the a function in all helper headers for the main file to compile everywhere, even if it's just a dummy function on ARM currently.